trunner/tools/gdb.py

```diff
@@ -151,6 +151,45 @@ def _close(self):
         self.logfile.close()
 
 
+def OpenocdProcess(
```

Comment:
TBH my intention was creating another class for simply calling openocd commands without a need for `with`, but the only difference is removing yield from a run or simply __call__ method in our new class. Probably we can create a parent class `Openocd` and making the `OpenocdGdbServer` a child class. @mateusz-bloch could you guide @maska989 how we can do it?

Reply:
Please also note that we want to use this new class in the flashing process of stm target.